tools/externaldriver/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+SET(MDAL_DUMMY_EXTERNAL_DRIVER
+    mdalDummyExternalDriver.cpp)
+	
+INCLUDE_DIRECTORIES(
+    ${CMAKE_CURRENT_SOURCE_DIR}
+    ${CMAKE_CURRENT_BINARY_DIR}
+    ${CMAKE_BINARY_DIR}
+    ${CMAKE_SOURCE_DIR}
+    ${CMAKE_SOURCE_DIR}
+)
```

Comment:
not sure if you need this at all. you need binary dirs only if you generate some headers, maybe at maximum ${CMAKE_SOURCE_DIR}